test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1

```diff
@@ -148,8 +148,8 @@ $testCases = @{
         HelpFiles            = "Microsoft.PowerShell.PSReadline.dll-help.xml"
```

Comment:
This should be `Microsoft.PowerShell.PSReadLine.dll-help.xml`.

Reply:
Will fix